react-native/react/local-debug.desktop.js

```diff
@@ -18,8 +18,8 @@ let config = {
   reduxDevToolsEnable: false,
   redirectOnLogout: true,
   reduxDevToolsSelect: state => state, // only watch a subset of the store
-  enableStoreLogging: false,
-  forwardLogs: false
+  enableStoreLogging: true,
+  forwardLogs: true
```

Comment:
It makes sense to enable this stuff otherwise the log is somewhat useless.

Also with store logging we can even re-enter the same state the user was in!